core/pta/attestation.c

```diff
@@ -186,17 +187,17 @@ static TEE_Result deserialize_key(uint8_t *buf, size_t buf_sz)
 	if (!sz)
 		goto out;
 	p += sz;
-	buf_sz -= sz;
+	sz_done += sz;
 	sz = deserialize_bignum(p, buf_sz, key->d);
```

Comment:
If `buf_sz` is not decremented, then the size check in `deserialize_bignum()` becomes useless, doesn't it?

Reply:
Yeah, I missed this check in `deserialize_bignum()`. I guess I can just remove the `assert()` in this `deserialize_key()`